mysql-plugin/src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -41,7 +42,12 @@ public static void overrideUserAndPasswordIfProvided() {
   }
 
   @Before(order = 1, value = "@MYSQL_SOURCE_TEST")
-  public static void setSelectQuery() {                
+  public static void setSelectQuery() {
+    String uuid = RandomStringUtils.randomAlphabetic(10);
+    PluginPropertyUtils.addPluginProp("sourceTable", PluginPropertyUtils.pluginProp("sourceTable").
```

Comment:
Refactor this logic into a separate function.

Reply:
Done